reboot_toolkit/toolkit_backend.py

```diff
@@ -335,7 +372,13 @@ def load_games_to_df_from_s3_paths(
                             f"{game_path}-v1-0-0", index_col=[0], use_threads=True
                         ).dropna(axis=1, how='all')
                 else:
-                    current_game = wr.s3.read_csv(game_path, index_col=[0], use_threads=True).dropna(axis=1, how='all')
+                    movement_filenames = wr.s3.list_objects(game_path)

```

Comment:
This sampling should also apply for the "if" block above where the user is selecting metrics files